tests/functional/test_install_vcs_svn.py

```diff
@@ -8,11 +8,11 @@
 @pytest.mark.network
 def test_obtain_should_recognize_auth_info_url(call_subprocess_mock, script):
     url = 'svn+http://username:password@svn.example.com/'
-    svn = Subversion()
+    svn = Subversion(use_interactive=False)
```

Comment:
I would add a comment (here and for the other one) saying you're passing `use_interactive=False` to check that the remote call options are being used.